client/api/python/heketi/heketi.py

```diff
@@ -54,3 +56,535 @@ def hello(self):
         r = requests.get(self.host + uri, headers=headers)
         return r.status_code == requests.codes.ok
 
+
+class Cluster(object):
+    """ Class to run cluster operations """
+
+    def __init__(self, client):
+        self.client = client
+
+    def create(self):
+        headers = {}
+        method = 'POST'
+        uri = "/clusters"
+
+        output = ''
+        err = ''
+
+        self.client._set_token_in_header(headers, method, uri)
+        r = requests.post(self.client.host + uri, headers=headers)
+
+        if r.status_code == requests.codes.created:
+            output = r.json()
+        else:
+            err = r.status_code
+
+        return output, err
```

Comment:
It seems that the output of each of these API functions is a dict.  That's ok, as long as it is documented, which reminds me, this needs documentation :).  I do not see the need to return the status code to the caller.  They probably want to know if it worked or not, which means most likely raising an exception.  I remember you mentioned this in the gitter chat, but now that I see the code, it seems clearer that an exception would need to be raised.  Here is an example from [OpenStack Swift python client](https://github.com/openstack/python-swiftclient/blob/master/swiftclient/client.py#L787).  It does not have to be that detailed, but I just wanted to show an example of it raising an exception when an incorrect HTTP code has been received.